### DISCUSSION TITLE
I deleted a few tags, and now other tags are missing.

### DISCUSSION THREAD

**Discussion by BujjuIsABee** [community/community#35360] [New to GitHub] [answered]
I deleted some tags in my repository, and now some other tags are missing. I noticed that I had to click the "I understand, delete this tag" button multiple times, but I don't know why this would delete different tags. Is there anything I can do to get them back?

**Comment by fadyat** [comment, marked as answer]
You may still have these tags in your local repository.
Try to find them locally and submit to github again.

**Reply by queenofcorgis** [reply]
Thanks for chiming in @fadyat - @BujjuIsDumb did this help you - if so don't forget to mark it as the answer 😄
